cluster/gce/manifests/etcd.manifest

```diff
@@ -61,13 +61,17 @@
       }
         ],
     "livenessProbe": {
-      "httpGet": {
-        "host": "127.0.0.1",
-        "port": {{ etcd_livenessprobe_port }},
-        "path": "/health"
+      "exec": {
+        "command": [
+          "/bin/sh",
+          "-c",
+          "exec /usr/local/bin/etcdctl --endpoints=127.0.0.1:{{ port }} {{ etcdctl_certs }} --command-timeout=15s endpoint health"
```

Comment:
Consider:
"set -x; exec ... " 
This gives handy log line about all the parameters supplied to the command.

Reply:
Done